### PR TITLE
feat: add collaboration and marketplace sections

### PR DESCRIPTION
**Pull request by sr-remsha**
Added a Collaboration section, which includes documentation for publication, sharing and notifications features in both API and AI DIAL Chat UI. 
Added docs for apps registration and DIAL Marketplace.

### CODE REVIEW

File: docs/tutorials/collaboration/2.enable-publications.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+In the response, you get an object with the `PENDING` status, which is awaiting the action from the admin: approve or reject. While your request is pending, you can delete it by calling [/v1/ops/publication/delete](#) endpoint.
+
+### Admin Flow
+
+As an administrator, you can get a list of publication requests awaiting your decision by calling [/v1/ops/publication/list ](https://epam-rail.com/dial_api#tag/Publications/paths/~1v1~1ops~1publication~1list/post) endpoint. Further, you can call [/v1/ops/publication/get](https://epam-rail.com/dial_api#tag/Publications/paths/~1v1~1ops~1publication~1delete/post) endpoint to get a specific publication request and then execute [/v1/ops/publication/approve](https://epam-rail.com/dial_api#tag/Publications/paths/~1v1~1ops~1publication~1approve/post) or [/v1/ops/publication/reject](https://epam-rail.com/dial_api#tag/Publications/paths/~1v1~1ops~1publication~1reject/post) to change its status.
```
Comment (astsiapanay) [inline docs/tutorials/collaboration/2.enable-publications.md:17]: to get publication -> https://epam-rail.com/dial_api#tag/Publications/paths/~1v1~1ops~1publication~1get/post

File: docs/tutorials/collaboration/3.sharing.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### Endpoints for resource owners
+
+* `/create`: resource owners can use this endpoint to share a resource by creating an `invitationLink` which includes an `invitationId`. On top of that, you can grant **read** or **write** access to your resource. **Note**: If an app/user changes the shared resource, HTTP **etag** is used by AI DIAL Core to determine versioning and resolve conflicts. You can use [Subscription API](/tutorials/collaboration/notifications#subscriptions) to get notifications about changes made by other users or applications. Refer to [DIAL API](https://epam-rail.com/dial_api#tag/Sharing/paths/~1v1~1ops~1resource~1share~1create/post) to view examples. **Limitations**: the system does not support getting all the recipients of a shared resource.
+* `/copy`: this endpoint is used when you need to modify a shared resource. For example, if you have a conversation with attachments that has been shared, and you decide to add another attachment, this endpoint ensures that other users can access the updated shared resource with the new attachment included.
```
Comment (astsiapanay) [inline docs/tutorials/collaboration/3.sharing.md:14]: you need to add permissions to users for a new resource.

Comment (sr-remsha) [reply docs/tutorials/collaboration/3.sharing.md]: fixed

**Review by astsiapanay — DISMISSED** [review]

File: docs/user-guide.md [hunk 3]
```diff
@@ -1124,11 +1102,24 @@ You can register and view your applications in the [My applications](#my-applica
 
 ![](./img/add-app.png)
 
+#### Converse
```
Comment (astsiapanay) [inline docs/user-guide.md:1126]: Conversations?

**Review by astsiapanay — DISMISSED** [review]

**Review by VolhaBazhkova — DISMISSED** [review]

**Review by YauheniyaH — APPROVED** [review]

**Review by VolhaBazhkova — APPROVED** [review]

**Review by YauheniyaH — APPROVED** [review]

**Pull request merged**